### ISSUE TITLE
Regression: Pylint 1.7.1 incorrectly reports no-member for `pyuv.fs.FSEvent` whereas Pylint 1.6.5 worked

### ISSUE THREAD

**Issue by frerich** [PyCQA/pylint#1467]
### Steps to reproduce
1. Create a Python script file with the contents
```python
""" Test program for pylint incorrectly recording missing member on module """
import pyuv


EVENT_LOOP = pyuv.Loop.default_loop()
EVENT = pyuv.fs.FSEvent(EVENT_LOOP)
EVENT.stop()
```
2. Execute the tests script using Python (I used version 3.6.0 but I suppose any Python version will do) to observe that the test runs successfully, generating no output:
```sh
$ python fsevent.py
$
```
3. Run `pylint` on the test script, whitelisting the `pyuv` plugin in order to expose the members of the module

### Current behavior
Pylint 1.6.5 does not report any errors:
```sh
$ pylint --version && pylint --extension-pkg-whitelist=pyuv fsevent.py
No config file found, using default configuration
pylint 1.6.5,
astroid 1.4.9
Python 3.6.0 (default, Mar 20 2017, 11:23:28)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
...
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

Pylint 1.7.1 reports missing members on the `pyuv` module:
```sh
$ pylint --version && pylint --extension-pkg-whitelist=pyuv fsevent.py
No config file found, using default configuration
pylint 1.7.1,
astroid 1.5.2
Python 3.6.0 (default, Mar 20 2017, 11:23:28)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
No config file found, using default configuration
************* Module fsevent
E:  6, 8: Instance of 'module' has no 'FSEvent' member (no-member)

---------------------------------------------------------------------
Your code has been rated at -2.50/10 (previous run: 10.00/10, -12.50)
```

### Expected behavior

Pylint 1.7.1 should behave like Pylint 1.6.5 in that it does not claim that there is no such member as `FSEvent` on the module.

### pylint --version output

```sh
$  pylint --version
No config file found, using default configuration
pylint 1.7.1,
astroid 1.5.2
Python 3.6.0 (default, Mar 20 2017, 11:23:28)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
$
```

**Comment by frerich** [comment]
This might be the same issue as in #1444 but since `--extension-pkg-whitelist` is involved here, I suspected that a separate ticket might be warranted.

**Comment by PCManticore** [comment]
Not sure why we'd say that `FSEvent` is not defined, but this can be solved nowadays by providing `--generated-members=FSEvent` to pylint.
